README.md

```diff
@@ -77,6 +55,20 @@ The generated documentation will be under `docs/_build/html/`. You can open the
 root of the documentation by opening `docs/_build/html/index.html` on your
 favorite browser.
 
+### Running the Coordinator locally
+
+To run the Coordinator on your local machine, use the command:
+
+```shell
+$ python xain_fl/grpc/cli.py -f test_array.npy
```

Comment:
Would be nice to know how to generate the .npy file from a ml model (at least for keras).

Reply:
i think it's a good idea but is it ok if we create a separate ticket for this because it's outside of the scope of this task?

Reply:
That's is a good idea

Reply:
https://xainag.atlassian.net/browse/XP-298. thanks for approving!